doc/api/api_changes.rst

```diff
@@ -154,8 +154,9 @@ Different exception types for undocumented options
   and ``right`` arguments.  :meth:`~matplotlib.axes.Axes.set_ylim` and the
   3D equivalents (e.g. :meth:`~mpl_toolkits.axes.Axes3D.set_zlim3d`) had a
   corresponding problem.
-  The ``_min`` and ``_max`` arguments are now deprecated, and a ``TypeError``
-  will be raised if they would override the earlier limit arguments.
+  A ``TypeError`` will be raised if they would override the earlier
```

Comment:
It's reasonable to modify the changes in 3.0 section of the 3.1 docs. Additionally, I would add an entry to the section "changes in 3.1". People typically do not read the changelog of past versions.